### DISCUSSION TITLE
NO FUNCIONAN LOS BOTONES

### DISCUSSION THREAD

**Discussion by JESUS1959** [community/community#142584] [Other Feature Feedback, Questions, & Ideas]
### Select Topic Area

Question

### Body

A VER SI ME PUEDEN AYUDAR, los botones en mi maquina laser me dejaron de funcionar, y al pulsarlos no se enciende el laser
![error grbl](https://github.com/user-attachments/assets/fe9ee0df-df6e-4b1c-993a-d407f17eb8fe)

**Comment by LiteBrite82** [comment]
Hi @JESUS1959 ,

Thanks for being a part of the GitHub Community, we're glad you're here!

It's possible that another GitHub user might have run into this same issue and can help, but the GitHub Community Discussions focuses primarily on topics related to GitHub itself or collaboration on project development and ideas. We want to make sure you’re getting the best support you can, but this space may not be the right place for this particular topic.

Best of luck!
